study.md

```diff
@@ -85,8 +88,7 @@ Using Hash methods and string interpolation in Ruby, create a string using
 conspiracy theories.".
 
 ```ruby
-# your answer here
-```
+"My name is #{Dale[:given_name]} and I'm an #{Dale[:occupation]} that enjoys #{Dale[:hobbies][1]}."
```

Comment:
Remember that case matters.  This will cause an error because `Dale` does not exist.